### DISCUSSION TITLE
比特幣是什麼

### DISCUSSION THREAD

**Discussion by Chiway1229** [1101-GS4719-C/Python-Community-GS4719-C#699] [General]
https://www.investors.tw/what-is-bitcoin/

近期時常聽到什麼 【比特幣觸底反彈】、【比特幣進入寒冬】
究竟什麼是比特幣呢，上面這篇文講解了一些比特幣的基礎理論
像比特幣是一種沒有中心機構的數位貨幣、不受任何國家或機構控管、是透過區塊形成的長鏈來儲存交易資料等等
依我的看法，我認為比特幣現在雖然進入了低谷期，不過隨著這項技術越來越廣為人知，其消失的可能性也在逐漸降低
只要人心依舊信任他的價值，就能有漲回來的曙光
有朝一日能夠成長回一年前一枚比特幣6萬美金的黃金時期(希望啦)
如果想知道更多細節，可以在9/22去區塊鏈研究社找danwan1027

**Comment by danwan1027** [comment]
好想了解比特幣和認識幣圈大佬！

**Reply by Yunhans** [reply]
加入汪汪社帶你認識比特犬

**Reply by williamyen0623** [reply]
> 好想了解比特幣和認識幣圈大佬！
ig搜dodo.bnb 和 procyon._.loto 
後者貌似是一個現任台大生 很厲害！

**Comment by Yunhans** [comment]
我希望我能買到便宜的顯卡

**Reply by williamyen0623** [reply]
> 我希望我能買到便宜的顯卡

虛擬貨幣景氣不好, 顯卡相對之前暴漲打了7.5折左右, 肯定可以的

**Comment by williamyen0623** [comment]
比特幣是最早出現的去中心化虛擬貨幣, 在運用上還是有許多缺陷, 例如需要耗費大量電力去達到高算力來維持區塊鏈的一致性, 擴展性不及以太幣, 目前似乎是依賴稀缺性的特點和信仰在支撐
